doc/source/train/train.rst

```diff
@@ -85,10 +85,10 @@ system. Let's take following simple examples:
 
     Now let's convert this to a distributed multi-worker training function!
 
-    First, update the training function code to use PyTorch's
-    ``DistributedDataParallel``. With Ray Train, you just pass in your distributed
-    data parallel code as as you would normally run it with
-    ``torch.distributed.launch``.
+    All you have to do is use the ``ray.train.torch.prepare`` utility function to
+    easily setup your model, data, optimizer, & loss for distributed training.
```

Comment:
This mentions optimizer and loss but isn't implemented (yet?).

Reply:
That's right not implemented yet. The description has been updated.